harness/determined/deploy/gcp/gcp.py

```diff
@@ -322,27 +322,31 @@ def run_command(command: List[str], env: Dict[str, str], cwd: Optional[str] = No
     subprocess.check_call(command, env=env, stdout=sys.stdout, cwd=cwd)
 
 
-def validate_gcp_credentials(configs: Dict) -> None:
-    vars_file_path = get_terraform_vars_file_path(configs)
-    # Try to load google credentials from terraform vars when present.
-    if os.path.exists(vars_file_path):
-        tf_vars = terraform_read_variables(vars_file_path)
-        set_gcp_credentials_env(tf_vars)
+def validate_gcp_credentials(configs: Optional[Dict] = None, keypath: Optional[str] = None) -> None:
```

Comment:
this is doing a couple of things that are not just validating gcp credentials. would describe it as a utility that loads and sets gcp credentials be more accurate?

Reply:
could you please add a doc string that clarifies the inputs and the function's usage?

Reply:
Done!

Reply:
thankyou